ecom-login/ecom-login.css

```diff
@@ -0,0 +1,35 @@
+:root{
+    --shadow: #cbd5e1;
+}
+
+.login-wrapper{
+    width: 30rem;
+    box-shadow: 0 0 8px var(--shadow);
+}
+
+.login-input{
+    padding: 0.5rem 5rem;
+    border-radius: 3px;
+    border: 1px solid var(--gray-color);
+}
+
+.login-input:focus{
+    outline: none;
+}
+
+.login-page-btn{
+    padding: 0.5rem 7rem;
+    flex-shrink: 0;
```

Comment:
use width instead of flex-shrink and decrease its width. It looks huge.

Reply:
ok got it